prac_07/project_management.py

```diff
@@ -0,0 +1,32 @@
+def main():
+    """Project management main program with menu options."""
+    projects = load_projects(FILENAME)
+    print(f"Loaded {len(projects)} projects from {FILENAME}")
+
+    is_running = True
+    while is_running:
+        print("\n- (L)oad projects\n- (S)ave projects\n- (D)isplay projects\n"
+              "- (F)ilter projects by date\n- (A)dd new project\n- (U)pdate project\n- (Q)uit")
+        choice = input(">>> ").lower()
+
+        if choice == 'l':
+            filename = input("Enter filename to load: ")
+            projects = load_projects(filename)
+        elif choice == 's':
+            filename = input("Enter filename to save: ")
+            save_projects(projects, filename)
+        elif choice == 'd':
+            display_projects(projects)

```

Comment:
display_projects() should be split into 2 different functions namely display_incomplete_projects() and display_complete_projects() to follow SRP and to be able to reuse the functions in other sections